tests/Load/cleanup.sh

```
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -e
+
+. ./tests/Load/config.sh
```

Shellcheck

[info] Not following: ./tests/Load/config.sh was not specified as input (see shellcheck -x).

(SC1091)